mtags/src/main/scala-3/scala/meta/internal/pc/AutoImports.scala

```diff
@@ -111,12 +111,17 @@ object AutoImports {
       }
     }
 
-    private def specifyOwnerEdit(symbol: Symbol, owner: String): l.TextEdit = {
-      val name = symbol.showName
-      new l.TextEdit(nameEditRange(name), s"$owner.$name")
+    private def specifyOwnerEdit(owner: String): l.TextEdit = {
```

Comment:
Could you use `ShortenedNames` here?

Reply:
I'm not sure. Probably not, at least in its existing state.

Reply:
Anyway, even plain imports currently are ignoring `ShortenedNames`. I will look at it. It seems it will require tweaking `AutoImports` implementation.